From 695d8d5793e86f7ad06b931e2418e02fefc74c09 Mon Sep 17 00:00:00 2001
From: Alexander Volkov <a.volkov@rusbitech.ru>
Date: Fri, 7 Oct 2016 14:17:45 +0300
Subject: [PATCH 15/16] Add missing PLUGIN_CLASS_NAMEs

It will create a separate imported target for each plugin
instead of a single Qt5Widgets_.cmake for an arbirary one.
An imported target can be used e.g. for reading the
location property of a plugin.

Also change PLUGIN_CLASS_NAME for the bb10style plugin to
match the class name.

Change-Id: I0bdf98c4d47ee9e33611cb35a2f05f41e1948c7f
Reviewed-by: J-P Nurmi <jpnurmi@qt.io>
---
 src/plugins/styles/bb10style/bb10style.pro   | 2 +-
 src/plugins/styles/cleanlooks/cleanlooks.pro | 1 +
 src/plugins/styles/gtk2/gtk2.pro             | 1 +
 src/plugins/styles/motif/motif.pro           | 1 +
 src/plugins/styles/plastique/plastique.pro   | 1 +
 5 files changed, 5 insertions(+), 1 deletion(-)

diff --git a/src/plugins/styles/bb10style/bb10style.pro b/src/plugins/styles/bb10style/bb10style.pro
index 5f97aa8..3184a5f 100644
--- a/src/plugins/styles/bb10style/bb10style.pro
+++ b/src/plugins/styles/bb10style/bb10style.pro
@@ -1,7 +1,7 @@
 TARGET = bb10styleplugin
 
 PLUGIN_TYPE = styles
-PLUGIN_CLASS_NAME = BlackBerry10StylePlugin
+PLUGIN_CLASS_NAME = QBB10StylePlugin
 load(qt_plugin)
 
 INCLUDEPATH += $$PWD
diff --git a/src/plugins/styles/cleanlooks/cleanlooks.pro b/src/plugins/styles/cleanlooks/cleanlooks.pro
index 08a385d..72900bb 100644
--- a/src/plugins/styles/cleanlooks/cleanlooks.pro
+++ b/src/plugins/styles/cleanlooks/cleanlooks.pro
@@ -1,5 +1,6 @@
 TARGET  = qcleanlooksstyle
 PLUGIN_TYPE = styles
+PLUGIN_CLASS_NAME = QCleanlooksStylePlugin
 load(qt_plugin)
 
 QT = core gui widgets
diff --git a/src/plugins/styles/gtk2/gtk2.pro b/src/plugins/styles/gtk2/gtk2.pro
index 0612132..3c6c141 100644
--- a/src/plugins/styles/gtk2/gtk2.pro
+++ b/src/plugins/styles/gtk2/gtk2.pro
@@ -1,5 +1,6 @@
 TARGET  = qgtk2style
 PLUGIN_TYPE = styles
+PLUGIN_CLASS_NAME = QGtk2StylePlugin
 load(qt_plugin)
 
 QT = core-private gui-private widgets-private
diff --git a/src/plugins/styles/motif/motif.pro b/src/plugins/styles/motif/motif.pro
index 1607424..ebf9d3b 100644
--- a/src/plugins/styles/motif/motif.pro
+++ b/src/plugins/styles/motif/motif.pro
@@ -1,5 +1,6 @@
 TARGET  = qmotifstyle
 PLUGIN_TYPE = styles
+PLUGIN_CLASS_NAME = QMotifStylePlugin
 load(qt_plugin)
 
 QT = core gui widgets
diff --git a/src/plugins/styles/plastique/plastique.pro b/src/plugins/styles/plastique/plastique.pro
index dee3bb6..5c86ba8 100644
--- a/src/plugins/styles/plastique/plastique.pro
+++ b/src/plugins/styles/plastique/plastique.pro
@@ -1,5 +1,6 @@
 TARGET  = qplastiquestyle
 PLUGIN_TYPE = styles
+PLUGIN_CLASS_NAME = QPlastiqueStylePlugin
 load(qt_plugin)
 
 QT = core core-private gui gui-private widgets
-- 
2.7.4

